server/dao/listing.js

```diff
@@ -8,9 +8,10 @@ class ListingDao {
     const {
       rows,
     } = await this.dbPool.query(
-      'INSERT INTO rentr_listing (userid, title, price, num_bedroom, num_bathroom, is_laundry_available, is_pet_allowed, is_parking_available, images, description) VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10) RETURNING id;',
+      'INSERT INTO rentr_listing (userid, is_available, title, price, num_bedroom, num_bathroom, is_laundry_available, is_pet_allowed, is_parking_available, images, description) VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11) RETURNING id;',
```

Comment:
Does `$11` in this line now refer to `description` and `$2` to `is_available`?

Reply:
Yes, you are correct. May be I can change other line to from 1 to 11 just for the consistency.